docs/src/misc.rst

```diff
@@ -640,6 +643,11 @@ API
         On Windows, setting `PRIORITY_HIGHEST` will only work for elevated user,
         for others it will be silently reduced to `PRIORITY_HIGH`.
```

Comment:
We should probably add a similar clarification on IBM i, as you are not allowed to change your priority unless you have `*JOBCTL` special authority (even to lower it!).